modules/administration/pages/image-management.adoc

```diff
@@ -28,10 +28,14 @@ The Kiwi build type is used to build system, virtual, and other images.
 // Needs checking
 // at later places to be fix accordingly
 The image store for the Kiwi build type is pre-defined as a file system directory in the [path]``srv-www`` volume.
-{productname} serves the image store over HTTPS from [literal]``https://<SERVER-FQDN>/os-images/``.
+//{productname} serves the image store over HTTPS from [literal]``https://<SERVER-FQDN>/os-images/``.
+
+The image files can be downloaded from https://MANAGER-HOST/os-images/ORGANIZATION-ID/FILE-NAME.
```

Comment:
maybe add a conditional to have MANAGER-HOST change into UYUNI-HOST automatically ?